sample-kotlin/src/main/java/com/fblitho/lithoktsample/bordereffects/DashPathEffectBorderSpec.kt

```diff
@@ -18,22 +18,26 @@ import com.facebook.litho.ComponentContext
 import com.facebook.litho.Row
 import com.facebook.litho.annotations.LayoutSpec
 import com.facebook.litho.annotations.OnCreateLayout
+import com.facebook.litho.kotlin.children
+import com.facebook.litho.kotlin.row
 import com.facebook.litho.widget.Text
 import com.facebook.yoga.YogaEdge
 
 @LayoutSpec
 object DashPathEffectBorderSpec {
 
   @OnCreateLayout
-  fun onCreateLayout(c: ComponentContext): Component =
-      Row.create(c)
-          .child(
-              Text.create(c).textSizeSp(20f).text("This component has a dash path effect applied"))
-          .border(
-              Border.create(c)
-                  .color(YogaEdge.ALL, NiceColor.BLUE)
-                  .widthDip(YogaEdge.ALL, 5)
-                  .dashEffect(floatArrayOf(10f, 5f), 0f)
-                  .build())
-          .build()
+  fun onCreateLayout(c: ComponentContext): Component = row(c) {
+    children {
+      text {
+        text("This component has a dash path effect applied")
+        textSizeSp(20f)
+      }
+    }
+    border(Border.create(c)
```

Comment:
It would be nice with an extension so that we could express this as:
```
border {
  color(YogaEdge.ALL, NiceColor.BLUE)
  widthDip(YogaEdge.ALL, 5)
  dashEffect(floatArrayOf(10f, 5f), 0f)
}
```